modules/Dockerfile.opc-twin

```diff
@@ -1,17 +1,31 @@
-FROM microsoft/dotnet:2.2-sdk AS build
```

Comment:
I assume this file can be deleted and the proper Docker.debug file to be used for debugging

Reply:
Yep.